src/cascadia/TerminalApp/TerminalTab.cpp

```diff
@@ -1460,7 +1460,16 @@ namespace winrt::TerminalApp::implementation
         // TabViewItem().Background() only sets the color of the tab background
         // when the TabViewItem is unselected. So we still need to set the other
         // properties ourselves.
-        TabViewItem().Background(deselectedTabBrush);
+        //
+        // GH#11294: DESPITE the fact that there's a Background() API that we
+        // could just call like:
+        //
+        //     TabViewItem().Background(deselectedTabBrush);
+        //
+        // We actually can't, because it will make the part of the tab that
+        // doesn't contain the text totally transparent to hit tests. So we
+        // actually _do_ still need to set TabViewItemHeaderBackground manually.
+        TabViewItem().Resources().Insert(winrt::box_value(L"TabViewItemHeaderBackground"), deselectedTabBrush);

```

Comment:
Are they planning to correct this on the MUX side too? I guess even if they do, this method will work, right?

Reply:
I've got a personal todo to follow up about all this, since clearly the background API does basically nothing that we want. This will be a part of that conversation 😉

Reply:
Sounds good.